.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
```

Comment:
Why are tests run against python 2.6 and 2.7 instead of just 2.7 as in the old travis script?